index.js

```diff
@@ -3700,6 +3700,52 @@
   S.pairs =
   def('pairs', {}, [$.StrMap(a), $.Array($.Pair($.String, a))], pairs);
 
+  //# insert :: String -> a -> StrMap a -> StrMap a
+  //.
+  //. Inserts a given key and value into a shallow clone of a given
+  //. string map. Overrides the value if a key already exists.
```

Comment:
I think we should replace <del>a</del> with <ins>the</ins> in several places. In the second sentence it's important that we write *the* key rather than *a* (any) key.

Reply:
Makes sense.